launcher/jest.config.js

```diff
@@ -6,9 +6,9 @@ module.exports = {
   ],
   roots: ["<rootDir>/src/", "<rootDir>/tests/"],
   // testEnvironment: "jest-environment-jsdom",
-  // testMatch: [
-  //   "**/?(*.)+(spec|test).[tj]s?(x)"
-  // ],
+  testMatch: [
+     "**/?(*.)+(spec|test|int).[tj]s?(x)"
```

Comment:
added `.int` as suffix to run tests separately because I did not found a way to seperate `int.test.js` from `test.js`